src/config/active-response.c

```diff
@@ -320,19 +281,17 @@ int ReadActiveResponses(XML_NODE node, void *d1, void *d2)
         ar_flag |= LOCAL_AR;
     }
 
-    /* Close shared file for active response */
+end:
```

Comment:
Be careful here because the old version didn't free any field of `tmp_ar` since it has to keep available in the active response list.

The resources should be freed only if the insertion didn't happen.